client/components/navbar.jsx

```diff
@@ -14,22 +14,52 @@ export default function NavBar() {
           </a>
           <a className="justify-content-end ms-5 left-icons"><SlHeart/></a>
           <a className="justify-content-end left-icons"><VscAccount/></a>
-          <a className="justify-content-end left-icons shopping-cart"><BsCart3/></a>
-          <form className="d-flex w-100">
-            <div className="input-group flex-grow-1">
-              <input type="text" className="form-control ms-3 border-end-0" placeholder="Search products"/>
-              <a className="input-group-text me-3 search-button"><BsSearch/></a>
-            </div>
-          </form>
+
+          <a className="justify-content-end left-icons shopping-cart px-0"><BsCart3/></a>
+
+          <ProductSearch/>
           <a className="d-none d-sm-block right-icons mx-3"><SlHeart /></a>
-          <a className="d-none d-sm-block navbar-text text-white">View Wishlist</a>
-          <a className="d-none d-sm-block right-icons mx-3"><VscAccount /></a>
-          <a className="d-none d-sm-block navbar-text text-white">Log In<br />My Account</a>
-          <a className="d-none d-sm-block right-icons shopping-cart mx-3"><BsCart3 /></a>
+          <a className="d-none d-sm-block navbar-text text-white mb-0">View Wishlist</a>
+          <a className="d-none d-sm-block right-icons mx-3"><VscAccount/></a>
+          <a className="d-none d-sm-block navbar-text text-white mb-0">Log In<br/>My Account</a>
+          <a className="d-none d-sm-block right-icons shopping-cart mx-3"><BsCart3/></a>
+          <div className="d-flex flex-column mt-3">
+            <span className="d-none d-sm-block badge bg-white text-dark text-center">0</span>
+            <p className="d-none d-sm-block text-white cart-value">$500</p>
+          </div>
         </nav>
 
       </div>
     </header>
 
   );
 }
+
+function ProductSearch() {
+  const [searchResults, setSearchResults] = useState([]);
+
+  function handleSearch(event) {
+    event.preventDefault();
+
+    fetch('/api/products')
```

Comment:
I'm guessing you'll want a separate endpoint for searching. This one just returns everything. Either a separate endpoint or pass the search term either in a query parameter or payload.